src/javascript/app/base/binary_pages.js

```diff
@@ -119,7 +120,7 @@ const pages_config = {
     realws                   : { module: RealAccOpening,             is_authenticated: true },
     redirect                 : { module: Redirect },
     regulation               : { module: Regulation },
-    reset_passwordws         : { module: ResetPassword,              not_authenticated: true },
+    reset_passwordws         : { module: ResetPassword,              is_authenticated: false },
```

Comment:
I don't think this `is_authenticated` is needed at all, right?

Reply:
Correct, you need to specify `is_authenticated: false`